### PR TITLE
#2208 Show only first row from description at collection

### LINKED ISSUE #2208: Show only first row from description at collection
Let's make it show only the first line from the description of the collection
Test at https://kodadot.xyz/rmrk/collection/54563af23dba417622-CAT
https://user-images.githubusercontent.com/5887929/152416447-a8365828-2643-4584-a36d-682cc6ca2b02.mov

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #2208
- [ ] 

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot 2022-02-04 at 7 39 00 PM

### CODE REVIEW

**Review by roiLeo — CHANGES_REQUESTED** [review]
Instead of adding a new wrapper I would just change have the `maxCharsWrapper` attribute which was originally designed for this purpose.
You could try to pass it as a @Prop()

**Comment by prachi00** [comment]
> maxCharsWrapper

@roiLeo **maxCharsWrapper** decided whether or not to show the wrapper, the actual height is decided through this css only
Screenshot 2022-02-04 at 7 52 42 PM

**Comment by roiLeo** [comment]
> @roiLeo **maxCharsWrapper** decided whether or not to show the wrapper, the actual height is decided through this css only

Oh yes, you're right. I forgot about that one...

**Comment by roiLeo** [comment]
So with your solution there will be no difference between row 2 and 3, wouldn't it be better to pass the height of the wrapper as a Prop?

**Comment by prachi00** [comment]
> So with your solution there will be no difference between row 2 and 3, wouldn't it be better to pass the height of the wrapper as a Prop?

you mean pass a whole class that has max height set?

**Comment by roiLeo** [comment]
> > So with your solution there will be no difference between row 2 and 3, wouldn't it be better to pass the height of the wrapper as a Prop?
> 
> you mean pass a whole class that has max height set?

I would see something like this:
- pass height as prop number
- keep default: 130
- then in wrapper `:style="{max-height: $theprop + 'px'}"`

**Comment by prachi00** [comment]
> > > So with your solution there will be no difference between row 2 and 3, wouldn't it be better to pass the height of the wrapper as a Prop?
> > 
> > 
> > you mean pass a whole class that has max height set?
> 
> I would see something like this:
> 
> * pass height as prop number
> * keep default: 130
> * then in wrapper `:style="{max-height: $theprop + 'px'}"`

yes sounds better, made the required changes

File: components/shared/collapse/DescriptionWrapper.vue [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <template>
   <div>
-    <div :class="{ 'description-wrapper': activeWrapper && hasWrapper }">
+    <div
+      :style="{ maxHeight: `${rowHeight}px` }"
+      :class="{
+        'description-wrapper': activeWrapper && hasWrapper,
+        'row-one': row === 1 && activeWrapper && hasWrapper,
+      }">
```
Comment (roiLeo) [inline components/shared/collapse/DescriptionWrapper.vue:8]:
```suggestion
    <div
      :style="[activeWrapper && hasWrapper ? { maxHeight: `${rowHeight}px` } : null]"
      :class="{'description-wrapper': activeWrapper && hasWrapper }">
```

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
nice! 

image

pay 100 usd

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
okay lol unrelated to this issue

image

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 184.32 USD/KSM ~ 0.543 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xce2edc6b1e2e6ab55425228086b377043dc12620c98d572771ed2c5eaa6f7dfb](https://kusama.subscan.io/extrinsic/0xce2edc6b1e2e6ab55425228086b377043dc12620c98d572771ed2c5eaa6f7dfb)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**